app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -3,31 +3,46 @@ import React from "react";
 import { MenuVertical } from "neetoicons";
 import { Avatar, Typography, Dropdown } from "neetoui";
 
-const renderAvatarWithNameRoleAndImage = (name, { role, imageUrl }) => (
+const renderAvatarWithNameRoleAndImage = (
+  name,
+  { firstName, lastName, role, imageUrl }
+) => (
   <div className="flex flex-row items-center">
     <Avatar
       className="mr-2"
       user={{
-        name,
+        name: `${firstName} ${lastName}`,
         imageUrl,
       }}
     />
     <div>
-      <Typography style="h4">{name}</Typography>
+      <Typography style="h4">{`${firstName} ${lastName}`}</Typography>
       <Typography style="body2">{role}</Typography>
     </div>
   </div>
 );
 
-const renderDropdownButton = (setShowDeleteAlert, setSelectedContactId, id) => (
+const handleDelete = ({ setShowDeleteAlert, setSelectedContactId, id }) => {
+  setShowDeleteAlert(true);
+  setSelectedContactId(id);
+};
+
+const renderDropdownButton = ({
+  setShowDeleteAlert,
+  setSelectedContactId,
+  id,
+}) => (
   <div>
     <Dropdown buttonStyle="text" icon={MenuVertical}>
       <li> Edit </li>
       <li
```

Comment:
Go through the Slack thread once again.

Reply:
I think you are talking about using the `MenuItem.Button` from `neetoUI`. But I am facing an issue here, that when I use `MenuItem.Button` instead of the normal `li` tag it is showing as undefined. I have destructured these subcomponents from the `Dropdown` component and still it returns as undefined. Could you look into these and let me know if I missed anything here?
<img width="610" alt="Screenshot 2022-09-22 at 6 39 55 PM" src="https://user-images.githubusercontent.com/57584818/191764373-bf8c5137-94b7-41d1-8669-8730e5d093ca.png">

Reply:
Refer:
- https://github.com/nancy-binani/wheel/pull/8#discussion_r977596571
- https://neetoui.onrender.com/?path=/docs/migration-guide-v3-to-v4--page

Even after properly upgrading to V4, if you are facing this issue, then lemme know.

Reply:
Upgrading to `neetoUI v4` solved the issue.